src/content/pages/faq.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: FAQ
+subtitle: Frequently Asked Questions about EuroPython
+---
+
+# FAQ
+
+## Milestones
+
+<ul  className="milestone-done">
+  <li>**12 December 2024 Call for Teams starts</li>
+  <li>**10 January 2025** [Call for Proposals (CfP)](/programme/cfp/) opens</li>
+  <li>**11 January 2025** [Speaker Mentorship Programme](/programme/mentorship/) sign-up opens</li>
+</ul>
+
+<ul className="milestone-todo">
+  <li>**21 January 2025:** Speaker Mentorship signup closes</li>
+  <li>**27 January 2025:** Call for Proposals (CfP) closes</li>
+  <li>**January/February** Financial Aid Programme starts</li>
+  <li>**January/February 2025** Community Voting and Proposal Review</li>
+  <li>**February/March 2025** Programme selection process</li>
+  <li>**March 2025** Ticket sales open!**</li>
+  <li>**March 2025** Visa Information Page opens</li>
+  <li>**01 April 2025** Session preview</li>
+  <li>**20 April 2025** Schedule published</li>
+  <li>**??? 2025** Financial Aid Programme closes</li>
```

Comment:
```suggestion
  <li>**TBC 2025** Financial Aid Programme closes</li>
```
??? is giving me build error. So wondering if it is simpler to replace it with TBC